app/i18n.ts

```diff
@@ -0,0 +1,37 @@
+import { createInstance } from "i18next";
+import { initReactI18next } from "react-i18next/initReactI18next";
+import resourcesToBackend from "i18next-resources-to-backend";
+import i18nConfig from "@/i18n.config";
+import {InitInternationalizationArgs} from "@/types";
+
+export default async function initInternationalization(args: InitInternationalizationArgs) {
```

Comment:
```suggestion
export default async function intlInit(args: InitInternationalizationArgs) {
```
Maybe just intlInit but it's fine

Reply:
renamed